packages/gatsby/src/bootstrap/graphql-runner.js

```diff
@@ -0,0 +1,50 @@
+const { graphql } = require(`graphql`)
+const stackTrace = require(`stack-trace`)
+const withResolverContext = require(`../schema/context`)
+const errorParser = require(`../query/error-parser`).default
+
+const createGraphqlRunner = (schema, reporter) => (query, context = {}) =>
+  graphql(
```

Comment:
we had `const schema = store.getState().schema` in graphqlRunner before. Unsure if we need to get the latest schema each time we run a graphql query. If not mistaken we don't modify the schema anymore.

Reply:
CC @freiksenet

Reply:
We do update schema once after createPages finished (to infer `SitePage.context`), I would leave this part as it was (getting schema from redux every time)

Reply:
What @pieh said.